src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -14,6 +15,7 @@ import * as express from "express";
 import * as path from "path";
 import * as admin from "firebase-admin";
 import * as bodyParser from "body-parser";
+import * as semver from "semver";
```

Comment:
We have to be *really* careful with pulling new dependencies into `functionsEmulatorRuntime` because of the way we spawn this.  That's why we have the `parseVersionString` method instead of using a `semver` library.